src/puppetlabs/puppetdb/cli/shovel.clj

```diff
@@ -0,0 +1,374 @@
+(ns puppetlabs.puppetdb.cli.shovel
```

Comment:
I'd probably put activemq or amq in the name, maybe `puppetlabs.puppetdb.cli.amq-migration`?